person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :id, :name, :age
+
+  def of_age?
+    @age > 18
+  end
```

Comment:
- [ ] This method should return true if @age is **greater or equal** to 18 and false otherwise. Actually, you are just handling the greater case. Let's fix that